general/Cargo.toml

```diff
@@ -18,7 +18,7 @@ ipnetwork = "0.14.0"
 itertools = "0.8.0"
 lazycell = "1.2.1"
 lazy_static = "1.3.0"
-maxminddb = "0.13.0"
+maxminddb = { version = "0.13.0", features = ["mmap"], optional = true }
```

Comment:
I don't quite understand this change. `maxminddb` is a required feature --at least the code does not compile without? Also, the `mmap` feature pulls in `memmap` in any case, so it doesn't matter that `memmap` is declared optional. This could all be unconditional, except for compiling to wasm, maybe.

Reply:
I think what you want is something like:

```toml
[features]
geoip = ["maxminddb", "memmap"]
default = ["geoip"]
```

And then compile a noop of `store::GeoIpLookup`, or not compile it at all without `feature = "geoip"`.